examples/UKCore-ServiceRequest-Lab-CReactiveProtein-Example.xml

```diff
@@ -0,0 +1,48 @@
+<ServiceRequest xmlns="http://hl7.org/fhir">
+	<id value="UKCore-ServiceRequest-Lab-CReactiveProtein-Example"/>
+	<status value="active"/>
+	<intent value="order"/>
+	<category>
+		<coding>
+			<system value="http://snomed.info/sct"/>
+			<code value="108252007"/>
+			<display value="Laboratory procedure"/>
+		</coding>
+	</category>
+	<code>
+		<coding>
+			<system value="http://snomed.info/sct"/>
+			<code value="135842001"/>
```

Comment:
Thats a normal SNOMED CT concept, not a UTL one

how about, 56801000237103 | Plasmablasts percent count in blood

Reply:
Ignore, Serum C is in the UTL refset, but is a normal cncept